### DISCUSSION TITLE
Vertical tables

### DISCUSSION THREAD

**Discussion by atumas-bananamilk** [mantinedev/mantine#7089] [Feature requests]
Can we get responsive tables rather than scrollable or anything else?
All rows should become columns at some breakpoint.

Large screen (all looking good):
![Screenshot 2024-11-06 at 18 24 46](https://github.com/user-attachments/assets/37a143f7-5a3d-4128-b03c-e9c631c174dc)

Small screen (big mess):
![Screenshot 2024-11-06 at 18 25 09](https://github.com/user-attachments/assets/3e1cab3b-a6f2-45b7-9056-4280643b8a5d)

**Comment by rtivital** [comment]
See https://mantine.dev/core/table/#scroll-container
I'm not aware of any universal ways to make responsive tables work well on mobile for generic use cases. Setting min width is the easiest approach

**Comment by atumas-bananamilk** [comment]
I see this has been added:
https://mantine.dev/core/table/#vertical-variant

**Reply by atumas-bananamilk** [reply]
@rtivital it's actually now possible add breakpoints since vertical tables are out.

**Reply by rtivital** [reply]
What do you mean? Why was it not possible before? It is doable. Just difficult to maintain, so it won't be added, at least soon.

**Reply by atumas-bananamilk** [reply]
I meant "vertical" tables when requesting this:
`"All rows should become columns at some breakpoint."`

This is good now.

**Reply by rtivital** [reply]
It is not really what this feature adds, it allows using different styles for cells, it does not flip the data

**Reply by atumas-bananamilk** [reply]
Ok true, just tried.
